src/installing-workflow/configuring-registry.md

```diff
@@ -11,57 +11,21 @@ Every component that relies on registry uses two inputs for configuration:
 1. Registry Location environment variable named `DEIS_REGISTRY_LOCATION`
 2. Access credentials stored as a Kubernetes secret named `registry-secret`
 
-The helm classic chart for Deis Workflow can be easily configured to connect Workflow components to off-cluster registry. Deis Workflow supports external registries which provide either short lived tokens which are valid only for a specified amount of time or long lived tokens(basic username/password) which are valid forever for authenticating to them. For those registries which provide short lived tokens for authentication Deis Workflow will generate and refresh them such that the deployed apps will only have access to the short lived tokens and not to the actual credentials for the registries.
+The helm chart for Deis Workflow can be easily configured to connect Workflow components to off-cluster registry. Deis Workflow supports external registries which provide either short lived tokens which are valid only for a specified amount of time or long lived tokens(basic username/password) which are valid forever for authenticating to them. For those registries which provide short lived tokens for authentication Deis Workflow will generate and refresh them such that the deployed apps will only have access to the short lived tokens and not to the actual credentials for the registries.
```

Comment:
(realizing none of this was actually modified in this PR, but...) 
1. could use a space between `lived tokens` and `(basic username/password)`
2. could use a comma between `...which provide short lived tokens for authentication` and `Deis Workflow will generate...`
3. properly, `short lived` should be `short-lived`